### DISCUSSION TITLE
How can we non-programmers contribute?

### DISCUSSION THREAD

**Discussion by nevillepark** [FossifyOrg/General-Discussion#69] [Q&A] [answered]
I am not a trained programmer (I'm barely an untrained programmer), but I appreciate the project and would like to help out in my spare time if I can. What are some things you could use help with? 

Some things I thought of: 

- documentation, e. g. a wiki 
- writing website copy
- translating
- finding typos in strings
- graphic design (not just icons, but graphics for app repos & social media)
- web design
- evaluating website & app accessibility
- testing development builds for bugs 
- for experienced users, answering people's questions on official channels

I would be happy to help with documentation; I think it's important to have static, public documents people can refer to, not just having to ask questions in a chat. I can also help with proofreading and editing.

**Comment by Aga-C** [comment, marked as answer]
You don't need to ask what to do or be assigned to everything, you can just start contributing to these things. Just read contribution guidelines here: https://github.com/FossifyOrg/General-Discussion. You have there every information you need, even if you're not a developer.

> writing website copy
> graphic design (not just icons, but graphics for app repos & social media)
> web design

Here's a project with the website: https://github.com/FossifyOrg/fossifyorg.github.io. Just ask here if current developers needs some help with texts. 

> translating

https://hosted.weblate.org/projects/fossify. Just remember to translate only to your native language.

> finding typos in strings

If you see something like this, see in contribution guidelines how to edit files and create a pull request.

> evaluating website & app accessibility
> testing development builds for bugs

You can test on yourself and if you see any bug or problem with accessibility, raise an issue in a proper repository

> for experienced users, answering people's questions on official channels

Just participate in discussions if you know the answers to what people are asking.

**Reply by nevillepark** [reply]
Thanks so much! I'll take a look at all of these.
